src/index.ts

```diff
@@ -1,7 +1,10 @@
 // export directly from the index file to work around some bugs with
 // typescript and ngc and "barrel" files - https://github.com/ng-packagr/ng-packagr/issues/195
-export * from './page-header/index';
+export { PageHeaderComponent, PageHeaderModule } from './page-header/base/index';
 export * from './table/index';
 
 // Org specific components
-export * from './sterling/index';
+export {
+	PageHeaderComponent as SterlingPageHeaderComponent,
```

Comment:
I think we can safely rename the component classes at the source in this case

Reply:
Updated